.gitmodules

```diff
@@ -1,27 +1,34 @@
 [submodule "balena-yocto-scripts"]
 	path = balena-yocto-scripts
 	url = https://github.com/balena-os/balena-yocto-scripts.git
+	branch = master
 [submodule "layers/poky"]
 	path = layers/poky
 	url = https://github.com/balena-os/poky
+	branch = kirkstone
 [submodule "layers/meta-openembedded"]
 	path = layers/meta-openembedded
 	url = https://github.com/openembedded/meta-openembedded/
+	branch = kirkstone
 [submodule "layers/meta-imx"]
 	path = layers/meta-imx
 	url = https://github.com/nxp-imx/meta-imx.git
```

Comment:
@acostach which branch should we use here?

Reply:
@klutchell  it's kirkstone but the NXP BSP uses a specific commit so it's probably a good idea to stay on that one. This is where I got the branches and commits where applicable for this repo:
- https://github.com/nxp-imx/imx-manifest/blob/imx-linux-kirkstone/imx-5.15.71-2.2.0.xml#L32
- https://github.com/compulab-yokneam/meta-bsp-imx9/blob/kirkstone-2.2.0/scripts/meta-bsp-imx9.xml#L8